### ISSUE TITLE
Micdoodle Core Crash 1.12.2

### ISSUE THREAD

**Issue by Preminenc3** [micdoodle8/MicdoodleCore#11]
https://pastebin.com/6pMa1HG7

**Comment by MJRLegends** [comment]
This is a Galacticraft crash not micdoodle8 core

**Comment by Preminenc3** [comment]
Oh my bad it said micdoodle so I thought that that was the reason.

**Comment by Preminenc3** [comment]
CraftTweaker is the only one that I have that edits recipes and I have removed some galacticraft items but I have not changed any recipes.

**Comment by MJRLegends** [comment]
Could be a Galacticraft Tweaker thing but all that mod is doing is removing and adding recipes using GC api hooks for recipes

**Comment by Preminenc3** [comment]
I going to set  "Must use GC's own space metals in recipes?" to true and see if that works and if not I will disable GC Tweaker and see if that works.

**Comment by Preminenc3** [comment]
Also, MJR will you be putting your GC Tweaker stuff on the [CraftTweaker Wiki?](http://crafttweaker.readthedocs.io/en/latest/#)

**Comment by radfast** [comment]
ok good plan.

Could be `GalacticraftTweaker`.  I don't know anything about this mod!  @MJRLegends who made it?

**Comment by radfast** [comment]
In coding terms the problem here is that `OreDictionary.getOres("ingotMeteoricIron")` here - in this modpack - is not returning a `List` of type `<ItemStack>` as it normally should.  Probably because one of these other mods has modified `OreDictionary`.  That's not something I can trace or fix.

**Comment by MJRLegends** [comment]
Yeah i cant say this would be a issue with my mods that i can think of im only using the ore dict or adding ore dict entries

**Comment by Preminenc3** [comment]
Setting "Must use GC's own space metals in recipes?" to true fixed the issue

**Comment by MJRLegends** [comment]
Please could you try without GalacticraftTweaker added and with "Must use GC's own space metals in recipes?" To false

**Comment by radfast** [comment]
@Preminenc3 I'm assuming the 5 minutes between each post is because that's how long it actually takes Minecraft to load with your crazy large modlist 👅

**Comment by MJRLegends** [comment]
ok it seems to crash with ExtraPlanets... but im not using any OreRecipeUpdatable recipes just normal forge recipes and compressor and circuit fabricator ? any ideas?

**Comment by MJRLegends** [comment]
Seems to still crash with all but forge normal crafting recipes disabled. i dont think i can claim this issue as mine

**Comment by MJRLegends** [comment]
These are my only two recipes using Meteoric Iron ingots
![](https://i.gyazo.com/c99d290a42e906c7fb23ac4171251f4d.png)
![](https://i.gyazo.com/50b0b1d9939ef8c7518d36aa1bba7f67.png)

**Comment by radfast** [comment]
I can now reproduce the issue here, so you guys don't need to do anything else ... I'm on it :)

(Reproducing without **Extra Planets** but with a handful of other mods including some CoFH mods and AE2, IC2 and Tinker's Construct).

Setting the config we talked about to `true` is the current solution.

**Comment by MJRLegends** [comment]
Nice glad you found the issue

**Comment by memeturtleREAL** [comment]
im on tlauncher and i cannot find galacticraft core so some1 link it to me and say how to open settings?

**Comment by memeturtleREAL** [comment]
well and tell how it would work cuz i wanna play
